File: Java/663.java
import java.util.Scanner;

/**
 *
 * 663 - Contando Desde el Cero - Iván - https://github.com/wildfireOfMine
 */
public class Main {

    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        int prueba = sc.nextInt();
        for (int i = 1; i <= prueba; i++) {
            int ano = sc.nextInt();
            if (ano > 0) {
                ano--;
            }
            System.out.println(ano);
        }
    }
}

File: C++/663.cpp
#include <iostream>

using namespace std;

// 663 - Contando desde el Cero - Iván - https://github.com/wildfireOfMine

int main()
{
    int loop = 0;
    cin >> loop;
    for(int i = 0; i<loop; i++) {
        int year = 0;
        cin >> year;
        if (year > 0) {
            year--;
        }
        cout << year << endl;
    }

}
